src/main/resources/com/linecorp/armeria/server/docs/assets/css/armeria.css

```diff
@@ -148,4 +157,15 @@ h2 {
 
 .debug-textarea {
   width: 100%;
+  max-width: 100%;
```

Comment:
No. Otherwise you can resize the width of the text area beyond its width, which makes the text area overlap with the response area.